definitions/contested/json/CaseEventToFields/CaseEventToFields-DFR-911-nonprod.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseEventID": "FR_uploadCaseFiles",
+    "CaseFieldID": "uploadCaseDocument",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
```

Comment:
same here, I think you could've left it on the CaseEventToFields.json instead since you created a -prod one